data/groupedbaryw.json

```diff
@@ -0,0 +1,15 @@
+[{"id":0,"type":132,"xval":"Aug 20","yval":74.928,"name":"Total Average","breakdown":"1634","breakdowntype":"YW","parentrefid":163,"parentreftype":135,"zval":1300},
```

Comment:
we can run inconsistent date data through a parser. That is what moment js is for I believe. it can give us the the desired output format from strings of varying syntax